CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ option(CAPSTONE_BUILD_CSTOOL "Build cstool" ON)
 option(CAPSTONE_USE_DEFAULT_ALLOC "Use default memory allocation functions" ON)
 option(CAPSTONE_ARCHITECTURE_DEFAULT "Whether architectures are enabled by default" ON)
 option(CAPSTONE_DEBUG "Whether to enable extra debug assertions" OFF)
-option(CAPSTONE_INSTALL "Generate install target" OFF)
```

Comment:
This was done like this on purpose to support `FetchContent` workflows without doing custom hackery on the cache.

Reply:
Regardless, the change seems unrelated to the goal of this pr, so it should not be part of it.